File: Hydrogene/Nougat/UIKit/IBOutlet/IBOutlet/RootViewController.cs
namespace CSharpiOSIBOutletDemo
{
	using UIKit;

	[IBObject]
	class RootViewController : UIViewController
	{

    [IBOutlet]
    public UILabel label;

		public override id init()
		{
			this = base.initWithNibName("RootViewController") bundle(null);
			if (this != null)
			{
				title = "CSharp iOS IBOutlet Demo";
			}
			return this;
		}

		public override void viewDidLoad()
		{
			base.viewDidLoad();
			this.label.textColor = UIColor.redColor;
      this.label.text = "IBOutlet Demo Text";
		}

	}
}

File: Oxygene/Nougat/UIKit/IBOutlet/IBOutlet/RootViewController.pas
namespace OxygeneiOSIBOutletDemo;

interface

uses
  UIKit;

type
  [IBObject]
  RootViewController = public class(UIViewController)
  private
  public
    method init: id; override;
    method viewDidLoad; override;
    [IBOutlet] label: UILabel;
  end;

implementation

method RootViewController.init: id;
begin
  self := inherited initWithNibName('RootViewController') bundle(nil);
  if assigned(self) then begin
    title := 'Oxygene iOS IBOutlet Demo';
  end;
  result := self;
end;

method RootViewController.viewDidLoad;
begin
  inherited viewDidLoad;
  self.label.textColor := UIColor.redColor;
  self.label.text := "IBOutlet Demo Text";
end;



end.
